File: documentation-examples-scala/src/scala/dev/typr/foundationssc/docs/duckdb/BitStringType.scala
package dev.typr.foundationssc.docs.duckdb
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*



@SuppressWarnings(Array("unused"))
object BitStringType:
  //start
  val bitType: DuckDbType[String] = DuckDbTypes.bit
  val bit8: DuckDbType[String] = DuckDbTypes.bit(8)  // BIT(8)
  //stop

File: documentation-examples-kotlin/src/kotlin/dev/typr/foundationskt/docs/duckdb/BitStringType.kt
package dev.typr.foundationskt.docs.duckdb

import dev.typr.foundationskt.*
import dev.typr.foundationskt.data.*

@Suppress("unused")
class BitStringType {
    //start
    val bitType: DuckDbType<String> = DuckDbTypes.bit
    val bit8: DuckDbType<String> = DuckDbTypes.bitOf(8)  // BIT(8)
    //stop
}
